<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Q3DWindow</class>
 <widget class="QMainWindow" name="Q3DWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>631</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Qgis2threejs Exporter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="Q3DView" name="webView" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>862</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuScene">
    <property name="title">
     <string>&amp;Scene</string>
    </property>
    <widget class="QMenu" name="menuLayer">
     <property name="title">
      <string>Add Layer</string>
     </property>
     <addaction name="actionAddPlane"/>
     <addaction name="actionAddPointCloudLayer"/>
    </widget>
    <addaction name="actionSceneSettings"/>
    <addaction name="menuLayer"/>
    <addaction name="separator"/>
    <addaction name="actionReload"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <widget class="QMenu" name="menuPanels">
     <property name="title">
      <string>Panels</string>
     </property>
    </widget>
    <addaction name="menuPanels"/>
    <addaction name="separator"/>
    <addaction name="actionAlwaysOnTop"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuSaveAs">
     <property name="title">
      <string>Save Scene As</string>
     </property>
     <addaction name="actionSaveAsImage"/>
     <addaction name="separator"/>
     <addaction name="actionSaveAsGLTF"/>
    </widget>
    <widget class="QMenu" name="menuExport_Settings">
     <property name="title">
      <string>Export Settings</string>
     </property>
     <addaction name="actionLoadSettings"/>
     <addaction name="actionSaveSettings"/>
     <addaction name="separator"/>
     <addaction name="actionClearSettings"/>
    </widget>
    <addaction name="actionExportToWeb"/>
    <addaction name="menuSaveAs"/>
    <addaction name="separator"/>
    <addaction name="menuExport_Settings"/>
    <addaction name="separator"/>
    <addaction name="actionPluginSettings"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
    <addaction name="actionUsage"/>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionHomePage"/>
    <addaction name="actionSendFeedback"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuCamera">
     <property name="title">
      <string>Camera</string>
     </property>
     <addaction name="actionPerspective"/>
     <addaction name="actionOrthographic"/>
    </widget>
    <widget class="QMenu" name="menuWidgets">
     <property name="title">
      <string>Widgets</string>
     </property>
     <addaction name="actionNavigationWidget"/>
     <addaction name="separator"/>
     <addaction name="actionNorthArrow"/>
     <addaction name="actionHeaderFooterLabel"/>
    </widget>
    <addaction name="menuCamera"/>
    <addaction name="menuWidgets"/>
    <addaction name="separator"/>
    <addaction name="actionResetCameraPosition"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuScene"/>
   <addaction name="menuView"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidgetLayers">
   <property name="windowTitle">
    <string>Layers</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="Q3DTreeView" name="treeView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetAnimation">
   <property name="windowTitle">
    <string>Animation</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="AnimationPanel" name="animationPanel"/>
  </widget>
  <widget class="QDockWidget" name="dockWidgetConsole">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Console</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QListWidget" name="listWidgetDebugView">
       <property name="contextMenuPolicy">
        <enum>Qt::ActionsContextMenu</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&gt;&gt;&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditInputBox"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionSaveAsSTL">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>STL (.stl)</string>
   </property>
  </action>
  <action name="actionSceneSettings">
   <property name="text">
    <string>Scene Settings...</string>
   </property>
  </action>
  <action name="actionPerspective">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Perspective</string>
   </property>
  </action>
  <action name="actionOrthographic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Orthographic</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>Reload</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionAlwaysOnTop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Always on Top</string>
   </property>
  </action>
  <action name="actionExportToWeb">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Export to Web...</string>
   </property>
  </action>
  <action name="actionSaveAsImage">
   <property name="text">
    <string>Image (.png)...</string>
   </property>
  </action>
  <action name="actionResetCameraPosition">
   <property name="text">
    <string>Reset Camera Position</string>
   </property>
   <property name="shortcut">
    <string>Shift+R</string>
   </property>
  </action>
  <action name="actionLayer_Panel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Layer Panel</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionPluginSettings">
   <property name="text">
    <string>Plugin Settings..</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>&amp;Help Contents</string>
   </property>
   <property name="toolTip">
    <string>Open plugin document in default browser.</string>
   </property>
  </action>
  <action name="actionHomePage">
   <property name="text">
    <string>Plugin Home Page</string>
   </property>
   <property name="toolTip">
    <string>Open plugin homepage in default browser.</string>
   </property>
  </action>
  <action name="actionSendFeedback">
   <property name="text">
    <string>Send Feedback</string>
   </property>
   <property name="toolTip">
    <string>Open plugin issue tracking system in default browser.</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Qgis2threejs Plugin...</string>
   </property>
   <property name="toolTip">
    <string>Display this plugin version.</string>
   </property>
  </action>
  <action name="actionSaveAsOBJ">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Wavefront (.obj)</string>
   </property>
  </action>
  <action name="actionSaveAsGLTF">
   <property name="text">
    <string>glTF (.gltf, .glb)...</string>
   </property>
  </action>
  <action name="actionClearSettings">
   <property name="text">
    <string>Clear...</string>
   </property>
  </action>
  <action name="actionConsoleCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionConsoleClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionNorthArrow">
   <property name="text">
    <string>North Arrow...</string>
   </property>
  </action>
  <action name="actionHeaderFooterLabel">
   <property name="text">
    <string>Header/Footer Labels...</string>
   </property>
  </action>
  <action name="actionSaveSettings">
   <property name="text">
    <string>Save...</string>
   </property>
  </action>
  <action name="actionLoadSettings">
   <property name="text">
    <string>Load...</string>
   </property>
  </action>
  <action name="actionAddPointCloudLayer">
   <property name="text">
    <string>Add Point Cloud Layer...</string>
   </property>
  </action>
  <action name="actionNavigationWidget">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Navigation Widget</string>
   </property>
  </action>
  <action name="actionAddPlane">
   <property name="text">
    <string>Add Flat Plane</string>
   </property>
  </action>
  <action name="actionUsage">
   <property name="text">
    <string>Usage of 3D Viewer</string>
   </property>
   <property name="toolTip">
    <string>Show usage of 3D viewer</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Q3DView</class>
   <extends>QWidget</extends>
   <header>Qgis2threejs.q3dview</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3DTreeView</class>
   <extends>QTreeView</extends>
   <header>Qgis2threejs.q3dtreeview</header>
  </customwidget>
  <customwidget>
   <class>AnimationPanel</class>
   <extends>QWidget</extends>
   <header>Qgis2threejs.keyframes</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>Q3DWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
